airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/JdbcDestination.java

```diff
@@ -53,7 +36,7 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(JdbcDestination.class);
 
   public JdbcDestination() {
-    super("org.postgresql.Driver", SQLDialect.POSTGRES, new ExtendedNameTransformer());
+    super("org.postgresql.Driver", new ExtendedNameTransformer(), DefaultSqlOperations::new);
```

Comment:
This way of passing a function instead of a real SQLOperations doesn't smell good. It will make testing a bit more complex and it will delay failure.